roles/generate-jenkins/templates/DOCUMENTATION.j2

```diff
@@ -38,34 +39,44 @@ This image is deprecated. We will not offer support for this image and it will n
+We utilise the docker manifest for multi-platform awareness. More information is available from docker [here](https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-2.md#manifest-list) and our announcement [here]({{ lsio_blog_url }}/2019/02/21/the-lsio-pipeline-project/).
 
-Simply pulling `lscr.io/{{ lsio_project_name_short }}/{{ project_name }}` should retrieve the correct image for your arch, but you can also pull specific arch images via tags.
+Simply pulling `lscr.io/{{ lsio_project_name_short }}/{{ project_name }}:{{ release_tag }}` should retrieve the correct image for your arch, but you can also pull specific arch images via tags.
 
 The architectures supported by this image are:
 
-| Architecture | Tag |
-| :----: | --- |
-{% for item in available_architectures %}
-| {{ item.arch }} | {{ item.tag }} |
-{% endfor %}
-{% if development_versions %}
+| Architecture | Available | Tag |
+| :----: | :----: | ---- |
+| x86-64 | {{ '✅ | amd64-\<version tag\>' if 'x86-64' in (available_architectures | map(attribute="arch") ) else '❌ |' }} |
+| arm64 | {{ '✅ | arm64v8-\<version tag\>' if 'arm64' in (available_architectures | map(attribute="arch") ) else '❌ |' }} |
+| armhf| {{ '✅ | arm32v7-\<version tag\>' if 'armhf' in (available_architectures | map(attribute="arch") ) else '❌ |' }} |
 
+{% if development_versions %}
 ## Version Tags
 
-This image provides various versions that are available via tags. `latest` tag usually provides the latest stable version. Others are considered under development and caution must be exercised when using them.
+This image provides various versions that are available via tags. Please read the descriptions carefully and exercise caution when using unstable or development tags.
 
-| Tag | Description |
-| :----: | --- |
+| Tag | Available | Description |
+| :----: | :----: |--- |
+| latest | {{ '✅ | ' + development_versions_items[(development_versions_items | map(attribute="tag")) == "latest"].desc if has_latest == true else '❌ |' }} |
 {% for item in development_versions_items %}
-| {{ item.tag }} | {{ item.desc }} |
+{% if item.tag != 'latest' %}
+| {{ item.tag }} | ✅ | {{ item.desc }} |
+{% endif %}
 {% endfor %}
+
 {% endif %}
 {% if app_setup_block_enabled %}
-
 ## Application Setup
 
 {{ app_setup_block | trim }}
+{% if ["CI_SSL='true'", "CI_SSL= 'true'" ,"CI_SSL = 'true'"]|select("in", repo_vars) %}
+
+### Strict reverse proxies
+
+This image uses a self-signed certificate by default. This naturally means the scheme is `https`.
+If you are using a reverse proxy which validates certificates, you need to [disable this check for the container](https://docs.linuxserver.io/faq#strict-proxy).
+{% endif %}
```

Comment:
This block is missing from the README template, is that intentional?

Reply:
Yes, as i dont want another reason for the readme to be truncated on dockerhub.